clustered/common/src/main/java/org/ehcache/clustered/common/messages/ServerStoreOpMessage.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.common.messages;
+
+import org.ehcache.clustered.common.store.Chain;
+
+import java.nio.ByteBuffer;
+
+/**
+ */
+public abstract class ServerStoreOpMessage extends EhcacheEntityMessage {
+  public enum ServerStoreOp {
+    GET,
+    GETANDAPPEND,
+    APPEND,
+    REPLACE
+  }
+
+  private final String cacheId;
```

Comment:
Do we really need to send the whole cacheId every time?  Even if we don't want to optimize this now we should probably record this inefficiency in an issue.